### PR TITLE
Feature: take care of indent top & bottom

### PR DESCRIPTION
**Pull request by mostafa-sakhiri**
https://contractlive.atlassian.net/browse/CPM-7000

### CODE REVIEW

File: tests/horizontaltextindent.js [hunk 1]
```diff
@@ -0,0 +1,88 @@
+import VirtualTestEditor from '@ckeditor/ckeditor5-core/tests/_utils/virtualtesteditor';
+import testUtils from '@ckeditor/ckeditor5-core/tests/_utils/utils';
+import { getData as getViewData } from '@ckeditor/ckeditor5-engine/src/dev-utils/view';
+import Paragraph from '@ckeditor/ckeditor5-paragraph/src/paragraph';
+
+import TextIndent from './../src/textindent';
+import Horizontaltextindent from './../src/horizontaltextindent';
```
Comment (jeromelebaron) [inline tests/horizontaltextindent.js]:
```suggestion
import HorizontalTextIndent from './../src/horizontaltextindent';
```

File: tests/horizontaltextindent.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+	testUtils.createSinonSandbox();
+
+	afterEach( () => {
+		if ( editor ) {
+			return editor.destroy();
+		}
+	} );
+
+	it( 'defines plugin name', () => {
+		expect( Horizontaltextindent.pluginName ).to.equal( 'HorizontalTextIndent' );
```
Comment (jeromelebaron) [inline tests/horizontaltextindent.js]:
```suggestion
		expect( HorizontalTextIndent.pluginName ).to.equal( 'HorizontalTextIndent' );
```

**Review by jeromelebaron — CHANGES_REQUESTED** [review]
Un changement important

File: tests/horizontaltextindent.js (hunk 1 above)
Comment (mostafa-sakhiri) [reply tests/horizontaltextindent.js]: Bien vu.

File: tests/horizontaltextindent.js (hunk 2 above)
Comment (mostafa-sakhiri) [reply tests/horizontaltextindent.js]: Bien vu.

**Comment by jeromelebaron** [comment]
Ok for me

**Review by jeromelebaron — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by thiberaw — APPROVED** [review]

**Review by jonathanpruvost — APPROVED** [review]

**Comment by jeromelebaron** [comment]
Merged with https://github.com/concordnow/ckeditor5-textindent/pull/15

**Pull request closed without merging**